internal/session/service_close_connections.go

```diff
@@ -0,0 +1,31 @@
+package session
+
+import (
+	"context"
+
+	"github.com/hashicorp/boundary/internal/errors"
+)
+
+// CloseConnection is a domain service function that:
+// * closes requested connections
+// * uses the sessionId of the connection to see if the session meets conditions for termination
+func CloseConnections(ctx context.Context, sessionRepoFn *Repository, connectionRepoFn *ConnectionRepository,
+	closeWiths []CloseWith, opt ...Option) ([]closeConnectionResp, error) {
```

Comment:
nitpick: it looks like `opt` is not used, this should either be removed or named `_` to note that it is not used.